docs/guides/Getting-Started/getting-started/marketplace-overview.md

```diff
@@ -0,0 +1,177 @@
+- **External Seller:** Is a store hosted on another platform, or that is not yet hosted on any platform, and which owns the products and is usually responsible for delivery to the consumer. A VTEX marketplace is interested in integrating with this external store, in order to offer its products in its showcase.
+
+## Configuring VTEX Marketplace
+
+To operate as a marketplace, it is necessary to configure your store according to the steps described in [Configuring VTEX Marketplace](https://help.vtex.com/en/tutorial/configuring-vtex-marketplace--7splyp5MqIyt2Iyz5jsNzb).
+
+The actions that can be performed via API are the following:
+
+- [Adding seller](#Adding-Sellers)
+- [Configuring seller selection at checkout](https://developers.vtex.com/docs/guides/checkout-overview)
+- [Configure storefront](https://developers.vtex.com/docs/guides/getting-started-3)
+- [Configure payments](https://developers.vtex.com/docs/guides/payments-integration-guide)
+
+## Integrating External Marketplaces or External Sellers
+
+The Marketplace Protocol is a set of API requests and definitions to help you integrate external sellers into a VTEX marketplace, as well as external marketplaces into VTEX sellers.
+
+- **External seller protocol:** If you are a VTEX marketplace and want to integrate offers from an external seller into your catalog, see our [External Seller Integration Guide](https://developers.vtex.com/docs/guides/external-seller-integration-guide).
+- **External marketplace protocol:** If you are an external marketplace that wants to integrate with a VTEX seller, see [External Marketplace Integration Guide](https://developers.vtex.com/docs/guides/external-marketplace-integration-guide) to learn how to develop a custom connector to integrate with the architecture and catalog of VTEX sellers.
+
+## <a name="Adding-Sellers"></a>1. Marketplace: Adding Sellers
+
+VTEX marketplaces that want to make products from sellers available in their store, it will be necessary to perform some processes. The first step is to find out if the seller you want to connect to is a VTEX seller or an external seller. For external sellers, verify the [External Seller Integration Guide](https://developers.vtex.com/docs/guides/external-seller-integration-guide) and for VTEX sellers, follow the steps below.
+
+### Adding a seller for the Seller Portal
+
+The [Seller Portal](https://help.vtex.com/en/tutorial/seller-portal-primeiros-passos-para-o-marketplace--6ccErY3mCcfoW0qGXf167) is an edition of the VTEX platform with essential functionalities available, so that external or VTEX sellers can quickly integrate into VTEX marketplaces. The Seller Portal has default version developed by VTEX and if the marketplace wants its own customized version, it can develop from the VTEX version following the [Seller Portal Edition App](https://developers.vtex.com/docs/guides/seller-portal-edition-app) guide.
```

Comment:
```suggestion
The [Seller Portal](https://help.vtex.com/en/tutorial/seller-portal-primeiros-passos-para-o-marketplace--6ccErY3mCcfoW0qGXf167) is an edition of the VTEX platform with essential functionalities available, so that external or VTEX sellers can quickly integrate into VTEX marketplaces. 

>ℹ️ The Seller Portal has default version developed by VTEX and if the marketplace wants its own customized version, it can develop from the VTEX version following the [Seller Portal Edition App](https://developers.vtex.com/docs/guides/seller-portal-edition-app) guide.
```